src/Libraries/Microsoft.Extensions.Telemetry/Tracing.Sampling/SamplingOptions.cs

```diff
@@ -11,22 +11,28 @@ namespace Microsoft.Extensions.Telemetry.Tracing;
 public class SamplingOptions
```

Comment:
@dariusclay FYI, I decided to keep sampling for now, since I wasn't sure about its future.